Deprecate `Button` props (#1712)
iTwin/stratakit: apps/website/src/content/docs/components/mui/Button.md

Modifications specific to `Button`:

> - The `disableElevation` and `disableFocusRipple` props are deprecated and should not be used.

Author: ```suggestion
- The `disableElevation` and `disableFocusRipple` props are not supported.
```
